src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -662,4 +670,8 @@ class ExternalDatabaseManagementService(
         return Predicates.equal(TABLE_ID_INDEX, tableId)
     }
 
+    private fun publishStagingTableSql(tableName: String): String {
+        return "ALTER TABLE $tableName SET SCHEMA TO $MATERIALIZED_VIEWS_SCHEMA"
```

Comment:
Yeah, I figured that was probably better to avoid the errors on republishing / stopping of the syncs on the staging table / possible confusion around which one to use